tests/foreman/destructive/test_clone.py

```diff
@@ -83,6 +93,17 @@ def test_positive_clone_backup(
     assert target_sat.cli.Service.stop().status == 0
     assert target_sat.cli.Service.disable().status == 0
 
+    # If --skip-pulp-data make sure you can rsync /var/lib/pulp over per BZ#2013776
+    if skip_pulp:
+        # Copying satellite pulp data to target RHEL
+        assert (
+            target_sat.execute(
+                f'sshpass -p "{SSH_PASS}" rsync -e "ssh -o StrictHostKeyChecking=no" --archive --partial --progress --compress '
+                f'/var/lib/pulp root@{sat_ready_rhel.hostname}:/var/lib/'
```

Comment:
The [docs](https://docs.redhat.com/en/documentation/red_hat_satellite/6.15/html/administering_red_hat_satellite/cloning_satellite_server#sec-Pulp_Data_Considerations) say:
```
# rsync --archive --partial --progress --compress \
/var/lib/pulp/ target_server.example.com:/var/lib/pulp/
```
So this should be:
```suggestion
                f'sshpass -p "{SSH_PASS}" rsync -e "ssh -o StrictHostKeyChecking=no" --archive --partial --progress --compress '
                f'/var/lib/pulp/ root@{sat_ready_rhel.hostname}:/var/lib/'
```